index.html

```diff
@@ -13,226 +13,328 @@
 
 <body>
 
-  <div class="header">
+  <div class="container">
 
-    <div class="time">
-      <img class="icons image1" src="images/Time.png" alt="time">
-    </div>
+    <header class="header">
 
-    <div class="connection">
-      <img class="icons image2" src="images/Cellular-Connection.png" alt="Cellular_Connection-path">
-      <img class="icons image3" src="images/Wifi.png" alt="wifi">
-      <img class="icons image4" src="images//Capacity.png" alt="Capacity">
-    </div>
+      <div class="logo">
+        <a href="#">MUKUNZI</a>
+      </div>
 
-  </div>
+      <nav>
+        <img class="vector" src="images/Vector.png" alt="vector">
+        <img class="vector" src="images/Vector.png" alt="vector">
+        <img class="vector" src="images/Vector.png" alt="vector">
+      </nav>
 
-  <div class="name1">
+    </header>
```

Comment:
[Optional] Please stick with the hamburger button presented in Figma design
![image](https://user-images.githubusercontent.com/74744787/158880562-a6e71c37-5b39-4261-b4b8-cff633984f1e.png)
![image](https://user-images.githubusercontent.com/74744787/158880605-fe9c035d-b5ca-4f60-aa88-586477b76f40.png)
you can download the union instead of just download vector
![image](https://user-images.githubusercontent.com/74744787/158880684-d6dd8de8-dce8-4d31-a8da-2b0480284e77.png)

Reply:
resolved 👍